apps/frontend/src/components/cards/StatusCardRow.vue

```diff
@@ -52,7 +76,7 @@ import {Filter} from '@/store/data_filters';
 import {Prop} from 'vue-property-decorator';
 
 interface CardProps {
-  icon: string;
+  icon?: string;
```

Comment:
This should no longer be necessary.

Reply:
It seems this is still used by the Passed/Failed/Not Applicable/Not Reviewed cards

Reply:
In that case I'd rather just pass the icon in as well for consistency.